composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "freeclimbapi/php-sdk",
-    "version": "4.3.0",
+    "version": "4.3.1",
     "description": "FreeClimb is a cloud-based application programming interface (API) that puts the power of the Vail platform in your hands. FreeClimb simplifies the process of creating applications that can use a full range of telephony features without requiring specialized or on-site telephony equipment. Using the FreeClimb REST API to write applications is easy! You have the option to use the language of your choice or hit the API directly. Your application can execute a command by issuing a RESTful request to the FreeClimb API. The base URL to send HTTP requests to the FreeClimb REST API is: /apiserver. FreeClimb authenticates and processes your request.",
     "keywords": [
         "openapitools",
```

Comment:
can we update license field to be 'MIT'?

Reply:
Hey @katebrune - This has been addressed in the latest push